doc/rules/control_structure/no_unneeded_control_parentheses.rst

```diff
@@ -12,7 +12,7 @@ Configuration
 
 List of control statements to fix.
 
-Allowed types: ``array``
+Allowed values: a subset of ``['break', 'clone', 'continue', 'echo_print', 'return', 'switch_case', 'yield', 'yield_from']``
```

Comment:
do we have a BC policy about this kind of thing? i suppose we call this one a bug fix? :trollface:

Reply:
I consider not properly validating user input a bug ;)  besides, if people have some configuration that is not in this set it won't do anything for them, now we'll let them know of that